shop.html

```diff
@@ -1 +1,127 @@
-
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Shop</title>
+  </head>
+  <body>
+    <header>
+      <label><a href="/">Shop</a></label>
```

Comment:
```suggestion
      <a href="/">Shop</a>
```

`<label>` используется в формах в сочетании с инпутами